### PR TITLE
[Stream] Unsubscription consequences - PSPEC-658

### PR DESCRIPTION
**Pull request by cascadingstyletrees**
As requested in PSPEC-658

### CODE REVIEW

File: products/stream/src/content/faq/index.md [hunk 1]
```diff
@@ -111,6 +111,10 @@ Resolution  |  Recommended bitrate
 
 </TableWrap>
 
+### If I cancel my stream subscription, are the videos deleted?
+
+Videos are removed If the subscription is not renewed within 30 days.
```
Comment (dcpena) [inline products/stream/src/content/faq/index.md]:
```suggestion
Videos are removed if the subscription is not renewed within 30 days.
```

File: products/stream/src/content/faq/index.md [hunk 2]
```diff
@@ -138,4 +142,4 @@ If your use case requires the limits to be increased, please contact support.
 
 ### How does Stream Live handle RTMP reconnections?
 
-As long as your streaming software reconnects, Stream Live will continue to ingest and stream your live video. Make sure the streaming software you are using to push RTMP feeds automatically reconnects if the connection breaks. Some apps like OBS reconnect automatically. Other apps like FFmpeg require custom configuration.
\ No newline at end of file
+As long as your streaming software reconnects, Stream Live will continue to ingest and stream your live video. Make sure the streaming software you are using to push RTMP feeds automatically reconnects if the connection breaks. Some apps like OBS reconnect automatically. Other apps like FFmpeg require custom configuration.
```
Comment (dcpena) [inline products/stream/src/content/faq/index.md]:
```suggestion
As long as your streaming software reconnects, Stream Live will continue to ingest and stream your live video. Make sure the streaming software you use to push RTMP feeds automatically reconnects if the connection breaks. Some apps like OBS reconnect automatically while other apps like FFmpeg require custom configuration.
```

**Review by dcpena — COMMENTED** [review]
Two tiny edits

File: products/stream/src/content/faq/index.md (hunk 2 above)
Comment (cascadingstyletrees) [reply products/stream/src/content/faq/index.md]: Those weren't by me, not sure what marked that as diff?  and thanks for catching it :)

File: products/stream/src/content/faq/index.md (hunk 1 above)
Comment (cascadingstyletrees) [reply products/stream/src/content/faq/index.md]: Thank you!

**Review by dcpena — APPROVED** [review]

**Pull request merged**